src/App.js

```diff
@@ -49,13 +50,18 @@ class App extends React.Component {
                         component={LoginPage}
                     />
                     <PrivateRoute
+                        exact
                         path='/clients/:clientId'
                         component={ClientDetailPage}
                     />
                     <PrivateRoute
                         path='/client/add'
                         component={AddClientPage}
                     />
+                    <PrivateRoute
+                        path='/clients/:clientId/payments/add'
+                        component={PaymentsAddPage}
```

Comment:
Suggestion: Change to `AddPaymentPage` since it is both consistent with `AddClientPage` and also just makes a bit more sense when reading it